client/worker.go

```diff
@@ -225,11 +224,11 @@ func (s *Session) sendFromQueueOrDecoy(loopSvc *utils.ServiceDescriptor) {
 func (s *Session) isDocValid(doc *pki.Document) error {
 	for _, provider := range doc.Providers {
 		_, ok := provider.Kaetzchen[constants.LoopService]
-		if !ok {
-			return errors.New("found a Provider which does not have the loop service")
+		if ok {
```

Comment:
should the client require a particular fraction of loop services to service provider nodes, or require that if a provider has kaetzchen that it must also have a loop service?

Reply:
that's out of scope for this feature addition